client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -31,37 +28,93 @@ export default () => {
 			select( 'automattic/verticals/templates' ).getTemplates( siteVertical.id )
 		) ?? [];
 
-	const homepageTemplates = templates.filter( template => template.category === 'home' );
+	const [ designs, otherTemplates ] = partition(
+		templates,
+		( { category } ) => category === 'home'
+	);
+
+	const [ selectedDesign, setSelectedDesign ] = useState< Template | undefined >();
+	const [ selectedLayouts, setSelectedLayouts ] = useState< Set< string > >( new Set() );
+	const resetLayouts = () => setSelectedLayouts( new Set() );
+	const toggleLayout = ( layout: Template ) =>
+		setSelectedLayouts( layouts => {
+			const nextLayouts = new Set( layouts );
+			if ( nextLayouts.has( layout.slug ) ) {
+				nextLayouts.delete( layout.slug );
+			} else {
+				nextLayouts.add( layout.slug );
+			}
+			return nextLayouts;
+		} );
+
+	const resetState = () => {
+		setSelectedDesign( undefined );
+		resetLayouts();
+	};
+
+	const transitionTiming = 250;
+	const hasSelectedDesign = !! selectedDesign;
+	const [ cp, setCp ] = useState< number >();
 
-	const [ selectedDesign, setSelectedDesign ] = useState< string | undefined >();
 	return (
-		<div className="design-selector">
-			<h1 className="design-selector__title">
-				{ NO__( 'Choose a starting design for your site' ) }
-			</h1>
-			<h2 className="design-selector__subtitle">
-				{ NO__( "You'll be able to customize your new site in hundreds of ways." ) }
-			</h2>
-			<div className="design-selector__grid">
-				{ homepageTemplates.map( template => (
-					<Card
-						className={ classnames( 'design-selector__design-option', {
-							'is-selected': template.slug === selectedDesign,
-						} ) }
-						key={ template.slug }
-						isElevated
-						onClick={ () => setSelectedDesign( template.slug ) }
-					>
-						<CardMedia>
-							<img
-								alt={ template.title }
-								src={ template.preview }
-								srcSet={ srcSet( template.preview, [ gridWidth / 2, gridWidth / 4 ] ) }
-							/>
-						</CardMedia>
-					</Card>
-				) ) }
+		<div className={ classnames( 'design-selector', { 'has-selected-design': selectedDesign } ) }>
+			<div
+				className="design-selector__header-container"
+				onClick={ () => resetState() }
+				aria-hidden={ hasSelectedDesign ? 'true' : undefined }
+			>
+				<h1 className="design-selector__title">
+					{ NO__( 'Choose a starting design for your site' ) }
+				</h1>
+				<h2 className="design-selector__subtitle">
+					{ NO__( "You'll be able to customize your new site in hundreds of ways." ) }
+				</h2>
 			</div>
+
+			<CSSTransition in={ ! hasSelectedDesign } timeout={ transitionTiming }>
+				<div
+					className="design-selector__grid-container"
+					onClick={ hasSelectedDesign ? resetState : null }
```

Comment:
According to the latest design there should be a button **<- Choose a different design** that should reset this state.